frontend/ui/widget/keyvaluepage.lua

```diff
@@ -272,7 +271,6 @@ function KeyValueItem:onShowKeyValue()
     return true
 end
 
-
 local KeyValuePage = FocusManager:new{
```

Comment:
Some of my double blank lines have some meaning. At least here and in the CoverBrowser and CalendarView plugins, I put 2 blank lines to separatate class names (1 blank line between KeyValueItem methods, 2 blank lines between the last KeyValueItem method and the first KeyValuePage stuff.

There's nothing really shocking having a look at all the removals- but I'm not fond of the principle of going cleaning other people mess (it's possible 90% of the blank lines are just random mess - but it's possible they had some meaning at the time of coding, to separate unrelated code/thoughts - which may now not make sense :) but I dunno, just an added double blank line would feel like a hint to me :) even if it's random.

(But I value more the blank lines I have put into functions bodies, to separate sections - Looks like I don't use 2 blank lines in there :) as I didn't see any after a quick look at this PR.)

Reply:
I'm not sure I left some around, but, yeah, I do agree with the double-LF in the main body being used as a separator being useful ;).

(Not applicable here, but some code formatting tools (*cough* clang-format *cough*) don't always agree with us on that point, so you sometimes have to make it explicit with a decorative comment).